packages/celotool/src/lib/helm_deploy.ts

```diff
@@ -817,11 +810,6 @@ async function helmParameters(celoEnv: string, useExistingGenesis: boolean) {
     `--set geth.gstorage_data_bucket=${fetchEnvOrFallback('GSTORAGE_DATA_BUCKET', '')}`,
     `--set geth.faultyValidators="${fetchEnvOrFallback('FAULTY_VALIDATORS', '0')}"`,
     `--set geth.faultyValidatorType="${fetchEnvOrFallback('FAULTY_VALIDATOR_TYPE', '0')}"`,
-    // Disable by default block age check in fullnode readinessProbe except for production envs
-    `--set geth.fullnodeCheckBlockAge=${fetchEnvOrFallback(
-      envVar.FULL_NODE_READINESS_CHECK_BLOCK_AGE,
-      `${isProduction()}`
-    )}`,
```

Comment:
Why are these disable blocks being removed?

Reply:
The `geth.fullnodeCheckBlockAge` helm parameter and equivalent `FULL_NODE_READINESS_CHECK_BLOCK_AGE` envVar allowed to enable/disable the age check in the old implementation for the readinessCheck. With the new implementation, as the node can be considered `ready` also when the block age is not recent, we can remove this config parameter and use the same readinessProbe for all deployments. 
The only place where this var was used was in line 391 in `packages/helm-charts/common/templates/_helpers.tpl` in the old implementation of the readinessProbe (removed in this PR too).